app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -192,9 +192,10 @@ paintTabs=Show tabs in page theme color
 tabsPerTabPage=Number of tabs per tab set:
 tabCloseAction=When closing an active tab:
 showTabPreviews=Show tab previews on hover
+showOpenedTabMatches=Show tab matches
 showHistoryMatches=Show history matches
 showBookmarkMatches=Show bookmark matches
-showOpenedTabMatches=Show tab matches
+showTopsiteMatches=Show top site matches
```

Comment:
while it follows this prefs pattern, wouldn't be better to say "show top site suggestions" instead?